### PR TITLE
new blog post

### PR DESCRIPTION
**Pull request by yabellini**
Blog post about the R-Ladies metameetup for useR! 2021

### CODE REVIEW

**Review by Athanasiamo — CHANGES_REQUESTED** [review]
The Rmd's wont be knit by the current workflow. So you will need to locally knit the Rmd to md and then add the md to git. In this case, since you dont have any code chunks, I'd recommend changing the file extension from Rmd to md, and that should be all that is needed :)

**Comment by yabellini** [comment]
Done! I change the file extension.  Thanks for this tip. I think we need to add these details to the contributing file.

**Review by Athanasiamo — APPROVED** [review]

**Comment by Athanasiamo** [comment]
Let me know when you think we should merge this @ChrisZasa  :)

File: content/post/2021-09-24-r-ladies-feat-user-2021/index.en.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+comment: no
+toc: no
+autoCollapseToc: no
+contentCopyright: no
+reward: no
+mathjax: no
+---
+
+![art_EN](https://user-images.githubusercontent.com/8397074/116813558-3e749e80-ab22-11eb-9c46-18fa17a3f1e7.png)
+
```
Comment (yabellini) [inline content/post/2021-09-24-r-ladies-feat-user-2021/index.en.md:24]: In this post, you will see that we have used the R-Ladies logo with the colors of the rainbow and the trans flags. We have also added a + to the R-Ladies name.
As symbols and words matter a lot, we wanted to explicitly communicate, with the use of these symbols, the importance of diversity and inclusion to our community.  We used them in this useR! to reinforce that we welcome diversity and put an effort into creating collectively a safe and friendly space.

**Review by yabellini — COMMENTED** [review]
I add the text about the logo and the plus in R-Ladies name.

**Comment by Athanasiamo** [comment]
I'll merge this today, if that is ok, @yabellini  :)

**Comment by yabellini** [comment]
Sure! Thanks!

**Comment by yabellini** [comment]
Mo, the texzt about the logo is not there??

**Pull request merged**